packages-exp/auth-exp/src/model/auth.d.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type AppName = string;
```

Comment:
I'm not personally a fan of aliasing simple types like this; it makes the interface less clear

`interface Config { apiKey: ApiKey }` provides less information than `interface Config { apiKey: string }`

Up to you though

Reply:
I couldn't find anything in the style guide regarding this, in general I think you're right, but for AppName, giving it a type helps disambiguate it since we almost refer it simple as 'name'